<launch>
    
    <param name="scan_line" type="int" value="32" />
    <param name="mapping_skip_frame" type="int" value="1" />
    <param name="map_pub_number" type="int" value="20" />
    <param name="loam_verbose_level" type="int" value="1" />

    <!-- if 1, do mapping 10 Hz, if 2, do mapping 5 Hz. Suggest to use 1, it will adjust frequence automaticlly -->
    <param name="mapping_skip_frame" type="int" value="1" />

    <!-- remove too closed points -->
    <param name="minimum_range" type="double" value="0.3"/>
    <param name="mapping_line_resolution" type="double" value="0.2"/>
    <param name="mapping_plane_resolution" type="double" value="0.4"/>

</launch>
